catalog.html

```diff
@@ -51,23 +56,23 @@
               <p>Пломбир с апельсиновым джемом</p>
               <p>1,5 кг &#215; <span>200 руб.</span></p>
               <p>300 руб.</p>
-              <button type="button">Удалить позицию из корзины</button>
+              <button type="hidden">Удалить позицию из корзины</button>
```

Comment:
```suggestion
              <button type="button">Удалить позицию из корзины</button>
```
А зачем исправили? Было ведь правильно. У кнопки нет такого типа, как `hidden`. https://webref.ru/html/button/type

Есть скрытые элементы `input` с таким типом.